forge/src/multi_runner.rs

```diff
@@ -114,18 +110,17 @@ impl MultiContractRunnerBuilder {
             },
         )?;
 
-        // add forge+sputnik specific contracts
-        known_contracts.insert("VM".to_string(), (HEVM_ABI.clone(), Vec::new()));
-        known_contracts.insert("VM_CONSOLE".to_string(), (HEVMCONSOLE_ABI.clone(), Vec::new()));
-        known_contracts.insert("CONSOLE".to_string(), (CONSOLE_ABI.clone(), Vec::new()));
+        // TODO Add forge specific contracts
+        //known_contracts.insert("VM".to_string(), (HEVM_ABI.clone(), Vec::new()));
+        //known_contracts.insert("VM_CONSOLE".to_string(), (CONSOLE_ABI.clone(), Vec::new()));
```

Comment:
I've also added a comment for this later, but if we're moving trace and log decoding to the CLI, we might not need to pass around `known_contracts` as we do now - the runner could just collect a mapping of `Address` -> `Bytes` that we can use to decode stuff later in the CLI. The CLI already has a mapping of all contracts to their ABI